src/utils/pagination.ts

```diff
@@ -0,0 +1,6 @@
+import i18n from '../translations/i18n';
+
+export const getTranslatedLabelForDisplayedRows = (from: number, to: number, count: number) =>
+  `${from}-${to === -1 ? count : to} ${i18n.t('common:of')} ${
+    count !== -1 ? count : `${i18n.t('common:more_than')} ${to}`
+  }`;
```

Comment:
tenkte egentlig interpolation: https://www.i18next.com/translation-function/interpolation#basic
men mulig det ikke blir noe finere med det heller da :P

Reply:
interpolation med conditionals?

Reply:
ja, blir jo noe ala: `i18next.t('key', { what: to === -1 ? count : to });`
men som sagt: vet ikke om det egentlig er noe bedre